plugins/mcreator-localization/help/default/entity/behaviour.md

```diff
@@ -1,5 +1,7 @@
 Living entity set to Mob will be able to attack, while a living entity set to Creature will be passive.
 
+A Raider is a monster, it will spawn in a raid.

```

Comment:
A Raider is a monster that spawns in a raid

Reply:
Fixed